gcloud-java-pubsub/src/main/java/com/google/gcloud/pubsub/spi/PublisherApi.java

```diff
@@ -0,0 +1,498 @@
+/*
```

Comment:
We had copied the license from gRPC. I think it makes more sense to use gcloud-java's license, so I'll update it.